### PR TITLE
Add AWS SAM and Fiber recipe 

### PR DESCRIPTION
**Pull request by the-exile-110**
Hi there, I'd like to submit a recipe for integrating AWS SAM with Go Fiber.   
This is a straightforward, out-of-the-box example of a SAM application designed for developers to get started with minimal setup. It provides a clear illustration of how to combine these technologies for rapid deployment on AWS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Here you can find the most **delicious** recipes to cook delicious meals using o
 ## 🌽 Table of contents
 
 - [Amazon Web Services (AWS) Elastic Beanstalk](/aws-eb)
```
Comment (gaby) [inline README.md:9]: Can you update the .github.com/dependabot.yml file? Thanks

**Review by gaby — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (the-exile-110) [reply README.md:9]: Thanks, I just updated the .github.com/dependabot.yml file.

**Comment by ReneWerner87** [comment]
@the-exile-110 sorry for the delay, can you refresh the branch so i can start the merge

**Comment by the-exile-110** [comment]
@ReneWerner87 
Hello, I didn't quite understand what you meant. Are you suggesting that I should delete the 'dependabot.yml' file?

**Comment by ReneWerner87** [comment]
yes, pls

**Comment by the-exile-110** [comment]
@ReneWerner87 
Thank you, I understand what you mean. I have just refreshed the branch.

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**